### PR TITLE
Search bar for contents

### LINKED ISSUE #131: Design and Implement Search on contents
On the Course consumption screen, it will be handy to be able to search on content names. 

- [x] Design and layout of the search UI (Bounty: 200)
- [ ] Implementation for Design (CSS, to done via motley) (Bounty: 500)
- [ ] Feature Implementation (Ember) (Bounty: 1000)

Note: No need to make separate API calls to get search results, assume all data to be loaded on the frontend only.

This is the course consumption screen:

screen shot 2018-06-19 at 8 12 04 pm

### PR DESCRIPTION
**Pull request by VibhorCodecianGupta**
@abhishek97 please review

Fixes #131, based on designs proposed in https://github.com/coding-blocks/motley/pull/232

### CODE REVIEW

File: app/pods/components/search-attempt/component.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  hideResultsBox = true;
+  qs = '';
+  @alias('searchTask.lastSuccessful.value') response
+  
+  @service router
+  @service store
+  
+  searchTask = task(function* () {
+    const searchQuery = yield this.get('qs')
+    const contents = this.get('store').peekAll('content');
```
Comment (abhishek97) [inline app/pods/components/search-attempt/component.js]: We only want to search this contents in this runAttempt

Comment (VibhorCodecianGupta) [reply app/pods/components/search-attempt/component.js]: Done @abhishek97

**Comment by VibhorCodecianGupta** [comment]
@abhishek97 made the changes

File: app/models/content.js [hunk 2]
```diff
@@ -25,5 +25,5 @@ export default DS.Model.extend({
     }
   }),
   progress: DS.belongsTo('progress'),
-  //section: DS.belongsTo('section')
+  section: DS.belongsTo('section')
```
Comment (abhishek97) [inline app/models/content.js]: This shall remain commented for the simple reason, a content may belong to many sections at a time.

**Comment by VibhorCodecianGupta** [comment]
@abhishek97 done, please review

File: app/models/content.js (hunk 2 above)
Comment (VibhorCodecianGupta) [reply app/models/content.js]: done @abhishek97

**Comment by VibhorCodecianGupta** [comment]
@abhishek97 any updates?

**Comment by abhishek97** [comment]
@VibhorCodecianGupta The search just doesn't work. Please test your code first.

**Pull request closed without merging**